core/condition.go

```diff
@@ -20,6 +20,11 @@ const (
 	// Values that could replace the placeholder: 127.0.0.1, 10.0.0.1, ...
 	IPPlaceHolder = "[IP]"
 
+	// DNSRCode is a place holder for DNS_RCODE
+	//
+	// Values that could be NOERROR, FORMERR, SERVFAIL, NXDOMAIN, NOTIMP and REFUSED
+	DNSRCode = "[DNS_RCODE]"
```

Comment:
I know the PR is still market as WIP, but well, here's an early review:

For the sake of consistency, this constant should be appended with `Placeholder`